layouts/partials/css.html

```diff
@@ -12,4 +12,4 @@
 {{ $prodCss      := $css | fingerprint }}
 <link rel="stylesheet" href="{{ $prodCss.RelPermalink }}" integrity="{{ $prodCss.Data.Integrity }}">
 {{ end }}
-<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.css" />
\ No newline at end of file
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/docsearch.js@2.6.3/dist/cdn/docsearch.min.css" />
```

Comment:
Are you claiming that this affects accessibility ratings?
Revert this change, `@2` is enough to get `@2.6.3`.

Reply:
there was some kind of error being shown in the console and when i updated to the latest version it went away, but i wonder that might have been due to some network issue because i am not getting that error again.
I'll revert this.